po/meson.build

```diff
@@ -0,0 +1 @@
+i18n.gettext(rvl_namespace, preset:'glib')
```

Comment:
Should be `rvl_name` instead of `rvl_namespace`

Reply:
Fixed both.